editor/editor_node.cpp

```diff
@@ -2607,7 +2607,7 @@ int EditorNode::_next_unsaved_scene(bool p_valid_filename, int p_start) {
 		if (!editor_data.get_edited_scene_root(i))
 			continue;
 		int current = editor_data.get_edited_scene();
-		bool unsaved = (i == current) ? saved_version != editor_data.get_undo_redo().get_version() : editor_data.get_scene_version(i) != 0;
+		bool unsaved = i == current ? saved_version != editor_data.get_undo_redo().get_version() : editor_data.get_scene_version(i) != 0;
```

Comment:
Maybe it's just me, and I understand that it shouldn't change anything, but it feels a little easier to read this line with the parenthesis.

Reply:
The reason for this change is that:
1. It is generally better to only have useful code
2. The parenthesis can be confusing because you are expecting that they do something when they don't

Reply:
>It is generally better to only have useful code

I mean, sure, but you should make that code easy to read too... And something that makes code easier to read could probably be considered useful anyways.

>The parenthesis can be confusing because you are expecting that they do something when they don't

Thing is, without them, I'm now confused about whether we're using `i == current` as the conditional, or `current` alone as the conditional. Even if you take into account that what we'll be using as the values are *also* conditionals, it's enough to make someone reread the line defining the `current` variable just to ensure they didn't misread a boolean as an integer.

Reply:
IMO, the parentheses make it more readable, especially when the ternary condition is being assigned to a variable (`=`)

`unsaved = i == current`

is less readable than

`bool unsaved = (i == current)`